sim/SIM-PROTOCOL.md

```diff
@@ -443,10 +523,41 @@ stat boosts are minor actions.
 > for messages from game mods that aren't supported by the client, like rule clauses 
 > such as Sleep Clause, or other metagames with custom messages for specific scenarios. 
 
-I'll document all the message types eventually, but for now this should be
-enough to get you started. You can watch the data sent and received from
-the server on a regular connection, or look at the client source code
-for a full list of message types.
+`|-combine`
+
+> A move has been combined with another (For example: Fire Pledge).
+
+`|-waiting|SOURCE|TARGET`
+
+> The `SOURCE` Pokémon has used a move and is waiting for the `TARGET` Pokémon
+> (For example: Fire Pledge).
```

Comment:
I think this is also deprecated for `|-activate|` of the relevant Pledge move.

Reply:
https://github.com/Zarel/Pokemon-Showdown/blame/master/data/moves.js#5137 (and elsewhere?)

Reply:
I guess the Pledge move messages are currently a huge hack that I need to go through and upgrade at some point.